### PR TITLE
chore(ReleaseInfo): update to 5.0.0

### PR DESCRIPTION
**Pull request by MaxTru**
* This is WIP as long as we don't have the final Modeler 5 feature set fixed.
* Please already review to ensure we have the major themes covered. Also feel free to add individual feedback already.

### CODE REVIEW

**Comment by andreasgeier** [comment]
What about plugin support for C8? If this is part of the release, I suggest to mention it in the first paragraph "Support for Camunda 8".

**Comment by andreasgeier** [comment]
Are the [improvements for Forms](https://github.com/camunda/camunda-modeler/issues/2541) worth an own bullet point?

**Comment by andreasgeier** [comment]
DMN for C8?

**Comment by MaxTru** [comment]
Thanks Andi, great hints! Will incorporate

**Comment by MaxTru** [comment]
Adjusted according to @andreasgeier comments. I did not include forms, since it is already packed and the changes are not so significant. Please feel free to challenge.

**Comment by andreasgeier** [comment]
Overall, I like the content. 
But I would suggest to change the title of the last bullet point to distinguish from first point, both are called "Support for Camunda 8". What about something like "Element template enhancements"?

Styling seems broken. Missing the marker, now it is a dot instead a dash. Also, the spacing between title and paragraph somehow increased on some list items and disappeared on others.

Old / new: different markers

Screen Shot 2022-04-01 at 12 55 13

Spacing too much or missing, different appearance per list item

Screen Shot 2022-04-01 at 13 01 48

**Comment by andreasgeier** [comment]
FYI: I checked styling by copy/paste the list to the modeler UI via browser inspector. You should double check if this is the cause of the styling issues.

**Comment by MaxTru** [comment]
> FYI: I checked styling by copy/paste the list to the modeler UI via browser inspector. You should double check if this is the cause of the styling issues.

Thanks, will check

**Comment by MaxTru** [comment]
> FYI: I checked styling by copy/paste the list to the modeler UI via browser inspector. You should double check if this is the cause of the styling issues.

I cannot reproduce.

When I use this in the Camunda Modeler, the styling is correct:
Screenshot_20220404_090925

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
...
       <ul className="dashed">
         <li>
-          <h4>Support for Camunda Cloud 1.4</h4>
-          With this release, we added support for Camunda Cloud 1.4.<br />
-          You can now evaluate a DMN decision in a Business Rules Task.
-          Specify the called decision (DMN) by its ID and assign the result to a variable via the properties panel.
-          Learn more about this feature in the <a href="https://docs.camunda.io/docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks/">Camunda Cloud docs on Business Rule Tasks.</a>
+          <h4>Support for Camunda 8</h4>
+          With this release, we added support for Camunda 8.<br />
+          You can now model BPMN, and DMN diagrams as well as Forms for Camunda 8.
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:49]: I think this comma is obsolete

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
...
-          With this release, we improved modeling of collapsed subprocesses for Camunda Platform.<br />
-          You can model collapsed subprocesses through the command pallet or by collapsing an expanded subprocess.
-          Use the blue drilldown icon to view and model the contents of the subprocess.
+          In Camunda 7 BPMN diagrams, you can now model collapsed subprocesses by
+          collapsing an expanded subprocess. Use the blue drilldown icon to view and model the contents of the subprocess.
         </li>
         <li>
-          <h4>Process Debugging for BPMN</h4>
-          The Modeler will now validate and show if BPMN elements are not supported in the target platform.
+          <h4>Camunda 8 support and new features for element templates</h4>
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:72]: As Andi already stated, "Camunda 8 support" is repeated here.

We could reorder the sections (move this below "Camunda 8 support") and rename it to "Powerful new features for element templates". But I would favor something more specific here

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 3]
```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
...
+          following our visions of a "Visual IDE".<br />
+          Learn more about the changes <a href="https://camunda.com/blog/2022/01/camunda-modeler-5-0-0-alpha-0-released/">in this blog post</a>.
+        </li>
+        <li>
+          <h4>Process Debugging for Camunda 8 BPMN diagrams</h4>
+          The Modeler will now support you when automating BPMN processes with Camunda 8 by showing implementation errors,
+          and giving hints how to fix them.
         </li>
         <li>
           <h4>Support drilldown into collapsed subprocesses</h4>
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:67]: Any further read on this here that we could link?

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 4]
```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
...
-          New welcome screen allows to easily create Platform and Cloud diagrams and forms.
-          The tab bar displays icons according to the file type and provides a new file menu via the plus button.
-          The properties panel has been completely redesigned. Tabs have been replaced with sections which can be opened and closed individually.
-          A granular breakdown with clear titles allows to identify relevant topics at a glance.
+          We re-worked the entire Modeler, including welcome screen, properties panel, status bar, tab bar, and bottom panel
+          following our visions of a "Visual IDE".<br />
+          Learn more about the changes <a href="https://camunda.com/blog/2022/01/camunda-modeler-5-0-0-alpha-0-released/">in this blog post</a>.
+        </li>
+        <li>
+          <h4>Process Debugging for Camunda 8 BPMN diagrams</h4>
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:62]: Any further read on this here that we could link?

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 1 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js:49]: true, thanks. Corrected

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 2 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js:72]: Corrected. Ty

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 4 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js:62]: Nope :-(

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 3 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js:67]: Just the blog post... (Which is alraedy linked per default from the whatsNewComponent)

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 4 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js:62]: Just the blog post... (Which is alraedy linked per default from the whatsNewComponent)

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 5]
```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
...
-          With this release, we improved modeling of collapsed subprocesses for Camunda Platform.<br />
-          You can model collapsed subprocesses through the command pallet or by collapsing an expanded subprocess.
-          Use the blue drilldown icon to view and model the contents of the subprocess.
+          In Camunda 7 BPMN diagrams, you can now model collapsed subprocesses by
+          collapsing an expanded subprocess. Use the blue drilldown icon to view and model the contents of the subprocess.
         </li>
         <li>
-          <h4>Process Debugging for BPMN</h4>
-          The Modeler will now validate and show if BPMN elements are not supported in the target platform.
+          <h4>New cusomization features for element templates</h4>
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:73]:
```suggestion
          <h4>New customization features for element templates</h4>
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 6]
```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
...
+          We added various new element template features: <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#groups">custom grouping</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/">documentation reference</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/">target element type</a>, and <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#icons">custom task icons</a>.<br />
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:74]:
```suggestion
          We added various new element templates features: <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#groups">custom grouping</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/">documentation reference</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/">target element type</a>, and <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#icons">custom task icons</a>.<br />
```

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 5 above)
Comment (barmac) [reply client/src/plugins/version-info/ReleaseInfo.js:73]: Perhaps it makes sense to check the text with [Grammarly](https://www.grammarly.com/). It takes just a few minutes but helps to improve writing a lot.

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 6 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js:74]: ty

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 5 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js:73]: will do. thanks

Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js]: Text now passed Grammarly :-)

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 7]
```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
...
         <li>
-          <h4>Process Debugging for BPMN</h4>
-          The Modeler will now validate and show if BPMN elements are not supported in the target platform.
+          <h4>Support drill down into collapsed subprocesses</h4>
+          In Camunda 7 BPMN diagrams, you can now model collapsed subprocesses by
+          collapsing an expanded subprocess. Use the blue drill down icon to view and model the contents of the subprocess.
+        </li>
+        <li>
+          <h4>New customization features for element templates</h4>
+          We added various new element templates features: <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#groups">custom grouping</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/">documentation reference</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/">target element type</a>, and <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#icons">custom task icons</a>.<br />
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:74]: The _target element type_ link does not have hash, so it links to the top of the docs.

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 8]
```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
...
         </li>
         <li>
           <h4>Improved overall UI to become a "Visual IDE"</h4>
-          New welcome screen allows to easily create Platform and Cloud diagrams and forms.
-          The tab bar displays icons according to the file type and provides a new file menu via the plus button.
-          The properties panel has been completely redesigned. Tabs have been replaced with sections which can be opened and closed individually.
-          A granular breakdown with clear titles allows to identify relevant topics at a glance.
+          We re-worked the entire Modeler, including welcome screen, properties panel, status bar, tab bar, and bottom panel
+          following our visions of a "Visual IDE".<br />
+          Learn more about the changes <a href="https://camunda.com/blog/2022/01/camunda-modeler-5-0-0-alpha-0-released/">in this blog post</a>.
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:60]: Trying to make the link more meaningful:

```suggestion
          <a href="https://camunda.com/blog/2022/01/camunda-modeler-5-0-0-alpha-0-released/">Learn more about the UI improvements</a>.
```

Based on https://accessibility.princeton.edu/how/content/links

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 7 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js:74]: True, but the reason is that `targetElementType` is explained not in a seperate section but as part of the overall "What is a template" concept.

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 8 above)
Comment (MaxTru) [reply client/src/plugins/version-info/ReleaseInfo.js]: :+1:

**Review by barmac — APPROVED** [review]

**Review by christian-konrad — APPROVED** [review]

**Pull request merged**